homework_10/FileLogger.java

```diff
@@ -0,0 +1,44 @@
+import enums.LoggingLevel;
+import exceptions.FileMaxSizeException;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger {
+    private final FileLoggerConfiguration config;
+
+    public FileLogger(FileLoggerConfiguration config) {
+        this.config = config;
+    }
+
+    public void writeInFile(String message, LoggingLevel loggingLevel) throws FileMaxSizeException, IOException {
+        File file = new File(config.getFileName());
+
+        if (file.exists() && file.length() >= config.getFileSizeMax()) {
```

Comment:
This `file.exists() && file.length() >= config.getFileSizeMax()` can be extracted to a separate method `isMaxFileSizeExceeded()` for the better readability